serializers/neos_serializer.rb

```diff
@@ -5,6 +5,7 @@ def daily(response)
       neos << {
         id: neo[:id],
         name: neo[:name],
+        event_date: neo[:close_approach_data][0][:close_approach_date],
```

Comment:
harbinger has many events
events have an event date which is separate from timestamps